larray/core/array.py

```diff
@@ -6335,43 +6337,88 @@ def combine_axes(self, axes=None, sep='_', wildcard=False):
+        a0_c0   b0   0   1
+        a0_c0   b1   4   5
+        a0_c1   b0   2   3
+        a0_c1   b1   6   7
+        a1_c0   b0   8   9
+        a1_c0   b1  12  13
+        a1_c1   b0  10  11
+        a1_c1   b1  14  15
+        >>> arr.combine_axes({('a', 'c'): 'ac'})
+           ac  b\\d  d0  d1
+        a0_c0   b0   0   1
+        a0_c0   b1   4   5
+        a0_c1   b0   2   3
+        a0_c1   b1   6   7
+        a1_c0   b0   8   9
+        a1_c0   b1  12  13
+        a1_c1   b0  10  11
+        a1_c1   b1  14  15
+        
+        # make several combinations at once
+        
+        >>> arr.combine_axes([('a', 'c'), ('b', 'd')])
+        a_c\\b_d  b0_d0  b0_d1  b1_d0  b1_d1
+          a0_c0      0      1      4      5
+          a0_c1      2      3      6      7
+          a1_c0      8      9     12     13
+          a1_c1     10     11     14     15
+        >>> arr.combine_axes({('a', 'c'): 'ac', ('b', 'd'): 'bd'})
+        ac\\bd  b0_d0  b0_d1  b1_d0  b1_d1
+        a0_c0      0      1      4      5
+        a0_c1      2      3      6      7
+        a1_c0      8      9     12     13
+        a1_c1     10     11     14     15
         """
-        axes = self.axes if axes is None else self.axes[axes]
-        # transpose all axes next to each other, using position of first axis
-        axes_indices = [self.axes.index(axis) for axis in axes]
-        min_axis_index = min(axes_indices)
-        transposed_axes = self.axes[:min_axis_index] + axes + self.axes
+        if axes is None:
+            axes = {tuple(self.axes): None}
+        elif isinstance(axes, AxisCollection):
+            axes = {tuple(axes): None}
+        elif isinstance(axes, (list, tuple)):
+            # checks for nested tuple/list
+            if all(isinstance(axis, (list, tuple, AxisCollection)) for axis in axes):
+                axes = {tuple(axes_to_combine): None for axes_to_combine in axes}
+            else:
+                axes = {tuple(axes): None}
+        # axes should be a dict at this time
+        assert isinstance(axes, dict)
+
+        transposed_axes = self.axes[:]
+        for axes_to_combine, name in axes.items():
+            # transpose all axes next to each other, using position of first axis
+            axes_to_combine = self.axes[axes_to_combine]
+            axes_indices = [transposed_axes.index(axis) for axis in axes_to_combine]
+            min_axis_index = min(axes_indices)
+            transposed_axes = transposed_axes - axes_to_combine
+            transposed_axes = transposed_axes[:min_axis_index] + axes_to_combine + transposed_axes[min_axis_index:]
         transposed = self.transpose(transposed_axes)
 
         new_axes = transposed.axes.combine_axes(axes, sep=sep, wildcard=wildcard)
         return transposed.reshape(new_axes)
 
-    def split_axis(self, axis, sep='_', names=None, regex=None):
-        """Split one axis and returns a new array
+    def split_axes(self, axes=None, sep='_', names=None, regex=None):
```

Comment:
We need to keep the old name for both methods (and have it display a warning...)